app/controllers/devise/two_step_verification_controller.rb

```diff
@@ -49,9 +57,12 @@ def create
     @otp_secret_key = params[:otp_secret_key]
     totp = ROTP::TOTP.new(@otp_secret_key)
     if totp.verify(params[:code])
-      current_user.update_attribute(:otp_secret_key, @otp_secret_key)
+      current_user.update_attributes(
+        otp_secret_key: @otp_secret_key,
+        require_2sv: false
```

Comment:
I don't think we need to set this here, it should only be set to false for deferrals?